docs/pages/guides/using-push-notifications-services.mdx

```diff
@@ -0,0 +1,106 @@
+---
+title: Using Push Notifications
+description: Learn about different push notification services.
```

Comment:
```suggestion
description: Learn about push notification services compatible with Expo and React Native apps.
```

Let's update this to make it more meaningful. Also, helps with SEO when we include keywords like React Native and/or Expo.

Reply:
Not sure of a perfect wording that does this, but do want to avoid the implication that these are the only options compatible with Expo apps. These just happen to be the ones that have purpose-built integrations (like config plugins).